owltools/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM maven:3.9.9-eclipse-temurin-23-noble
+
+LABEL maintainer="Microbiome Informatics Team www.ebi.ac.uk/metagenomics"
+
+LABEL version="1"
+LABEL software="owltools"
+LABEL software.version="2024-06-12"
+
+RUN apt install git
+
+RUN git clone https://github.com/owlcollab/owltools.git && \
```

Comment:
No need to clone and build it, they provide an execiutable ready to use: `https://github.com/owlcollab/owltools/releases/download/2024-06-12/owltools`

Reply:
ah I thought there was always a benefit to building it from source in containers, this will make it easier

Reply:
Well, this is more of a philosophical point. I think we should make our lives easier. More often than not, compiling stuff takes time and effort for little gain (a performance bump in some particular cases, but negligible). The bigger downside of building/compiling is that the containers become large, as all the required building tools take up storage that could be saved by simply pulling the binary/JAR.

Reply:
Good stuff. Worth mentioning [multi-stage builds[(https://docs.docker.com/build/building/multi-stage/) for any future cases where a binary does not exist, to still end up with a small image that doesn't include the build-tools used for compilation.

Reply:
ah I didn't know of this!! very cool, thanks Sandy!